src/plugins/data/public/query/query_string/language_service/types.ts

```diff
@@ -66,4 +67,7 @@ export interface LanguageConfig {
   supportedAppNames?: string[];
   hideDatePicker?: boolean;
   sampleQueries?: SampleQuery[];
+  addFiltersToQuery?: (query: string, filters: Filter[]) => string;
```

Comment:
query throughout the data plugin refers to an object. here we are going with string why? this conflicts with other functions in this same file reference query as an object making it inconsistent when registering a new language

Reply:
@joshuali925

Reply:
because this is the editor query `const editorQuery = useEditorQueryText();`, the editor query text is not a query object